aavealarm/network.tsx

```diff
@@ -103,6 +104,10 @@ function getAaveMarket(
       market = markets.AaveV3Metis;
       chainId = ChainId.metis_andromeda;
       break;
+    case chain == Chain.BASE && aaveVersion == 3:
+        market = markets.AaveV3Base;
```

Comment:
I guess it should be `markets.AaveV3Basenet` here?

Reply:
https://github.com/bgd-labs/aave-address-book/blob/50b9acc87adc355d669895bdfc884a79fd4b9abe/src/ts/AaveAddressBook.ts#L28C13-L28C25 mention and export AaveV3Base

Change log [1.31.7](https://github.com/bgd-labs/aave-address-book/compare/v1.31.6...v1.31.7) change chain name from Basenet to Base so I assume Base was the correct name

Reply:
Sounds quite reasonable! But then `package.json` should be updated. Currently it contains `"@bgd-labs/aave-address-book": "^1.31.4"` (an older version).

Reply:
Package have been updated